audit/src/main/scala/tech/beshu/ror/audit/instances/DefaultAuditLogSerializer.scala

```diff
@@ -16,6 +16,22 @@
  */
 package tech.beshu.ror.audit.instances
 
-import tech.beshu.ror.audit.AuditEnvironmentContext
+/**
+ * Serializer for audit events that is aware of **rule-defined verbosity**.
+ *
+ * - Includes `CommonFields` and `EsEnvironmentFields`.
```

Comment:
CommonFields, EsEnvironmentFields, ... are implementation details. Let's list all the fields in the description (with one line explanation if possible)